src/rev_ai/apiclient.py

```diff
@@ -324,6 +325,18 @@ def get_account(self):
         """
         url = urljoin(self.base_url, 'account')
         response = self.session.get(url)
-        response.raise_for_status()
+        self._handle_api_failure(response)
 
         return Account.from_json(response.json())
+
+    def _handle_api_failure(self, response):
+        """Helper function to provide more detailed error messages for the user
+        
+        :param response: response from the HTTP request
+        :raises: HTTPError
+        """
+        try:
+            response.raise_for_status()
+        except Exception as err:
```

Comment:
Prefer to use as specific exceptions as possible. Reasons: code-as-documentation and better clarity in logs in case of other kind of exception.